scripts/h5peditor-image-popup.js

```diff
@@ -295,33 +274,27 @@ H5PEditor.ImageEditingPopup = (function ($, EventDispatcher) {
     this.show = function (offset, imageSrc) {
       H5P.$body.get(0).appendChild(background);
       background.classList.remove('hidden');
-      setDarkroomDimensions();
+      setCropperDimensions();
       background.classList.add('hidden');
       if (imageSrc) {
         // Load image editing scripts dynamically
         if (!scriptsLoaded) {
-          H5P.setSource(editingImage, imageSrc, H5PEditor.contentId);
-          loadScripts();
+          loadScripts(() => self.setImage(imageSrc));
         }
         else {
           self.setImage(imageSrc);
         }
-
         if (offset) {
           var imageLoaded = function () {
             this.adjustPopupOffset(offset);
             editingImage.removeEventListener('load', imageLoaded);
           }.bind(this);
-
           editingImage.addEventListener('load', imageLoaded);
         }
       }
-      else {
-        H5P.$body.get(0).classList.add('h5p-editor-image-popup');
-        background.classList.remove('hidden');
-        self.trigger('initialized');
-      }
-
+      H5P.$body.get(0).classList.add('h5p-editor-image-popup');
+      background.classList.remove('hidden');
+      self.trigger('initialized');
```

Comment:
Removing this else causes the popup to load (with a message) after opening, in contrast to showing a loading symbol on the button and waiting to open until loaded. This means that if the user has already done some changes and wants to open the popup again, they can start changing things (e.g. rotating, cropping), then have it reset when the loading finishes. Did you mean to do this?